docs/content/reference/aggregated-docs.md

```
@@ -9635,6 +9635,333 @@ This section of the HyperShift documentation contains pages related to troublesh
+3. **Runs Claude Code** with the `/jira-solve` command to analyze each issue and implement a fix
+4. **Creates draft PR** from the `hypershift-community/hypershift` fork to `openshift/hypershift`
+5. **Updates Jira** after successful processing:
+   - Adds `agent-processed` label
+   - Transitions to "ASSIGNED" (OCPBUGS) or "Code Review" (CNTRLPLANE)
+   - Sets assignee to `hypershift-automation`
+
+### JQL Query
+
+Issues are selected for processing using this query:
+
+```sql
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)